### DISCUSSION TITLE
Synchronization Between ROG Strix Mouse Controls and G Helper

### DISCUSSION THREAD

**Discussion by SLNinjaXXX** [seerge/g-helper#3498] [General]
In the current setup, it seems that the ROG Strix Impact mouse requires re-synchronization with the G Helper software after every restart or shutdown. For example, DPI settings, Polling Rate and lighting configurations may not persist seamlessly between sessions, as shown in the screenshot.

Why do you think this synchronization issue occurs?

Could this be related to a software compatibility problem, a missing startup configuration, or a limitation in how the profiles are saved or loaded?

What potential solutions or workarounds could ensure that the settings persist across reboots?

Are there any known updates or patches for G Helper or the ROG Strix software that address this issue?

Feel free to share your insights or any relevant experiences with similar peripherals and software!

![Screenshot 2024-12-20 092644](https://github.com/user-attachments/assets/3a1e835a-2fdf-4a3b-9094-349fed6634fe)

**Comment by seerge** [comment]
@SLNinjaXXX hello,

This is because Strix Impact is very old (from 2017) and doesn't have any onboard memory unlike any other Asus mice. So it simply doesn't remember settings on reboot. Please check [this topic](https://github.com/seerge/g-helper/issues/3171) for more detailed explanation.

Since it's literally the only mouse that doesn't remember settings (while others do) - I'm currently not sure if it's worth to introduce a lot of extra logic in the app, just to force settings every time :)

You can use existing export / import settings feature tho.
